core/tests/test_state_transition_model.py

```diff
@@ -20,5 +27,149 @@ def test_create_incident_report_creates_new_state_transition(self):
             police_abstract_obtained="Yes",
             submitted_by=self.user,
         )
-        self.assertEqual(AssetIncidentReport.objects.count(), incident_report_count + 1)
-        self.assertEqual(StateTransition.objects.count(), state_transition_count + 1)
+        self.assertEqual(
+            apps.get_model("core", "AssetIncidentReport").objects.count(),
+            incident_report_count + 1,
+        )
+        self.assertEqual(
+            apps.get_model("core", "StateTransition").objects.count(),
+            state_transition_count + 1,
+        )
+
+
+class TestTransitionStateUpdateFromAssetStatusModification(CoreBaseTestCase):
+    """
+    Test updating of transition state when Asset status is modified
+    Update the transition state to CLOSED when an Asset status is changed from (DAMAGED,LOST) to
+    either (AVAILABLE,ALLOCATED)
```

Comment:
It should be that transition state is CLOSED when AssetStatus changes from ALLOCATED to either DAMAGED, LOST or AVAILABLE

Reply:
Noted. The implementation has been updated 👍